src/styles/main.scss

```diff
@@ -1,3 +1,51 @@
+@import "./vars.scss";
+
 body {
   margin: 0;
 }
```

Comment:
move this to other file, this is the file of block, it should be called calendar.scss, 3'rd point in checklist

Reply:
done, see 'v3'